guides/common/modules/proc_configuring-smart-proxy-to-provision-secure-boot-enabled-hosts.adoc

```diff
@@ -0,0 +1,105 @@
+[id="configuring-{smart-proxy-context}-to-provision-{client-os-context}-on-Secure-Boot-enabled-hosts"]
+= Configuring {SmartProxy} to provision {client-os} on Secure Boot enabled hosts
```

Comment:
```suggestion
= Configuring {SmartProxy} to provision {client-os} on Secure Booted hosts
```
Would this be acceptable? It would make the title shorter.

Reply:
We should leave this as is. Hosts with enabled "Secure Boot" are "securely booted", that's right. But "Secure Boot" is the proper name for the mechanism and the protocol used to achieve this and the term is part of the [UEFI specification](https://uefi.org/sites/default/files/resources/UEFI_Spec_2_10_Aug29.pdf) (see page 1411).

But i have to admit the title is pretty long :see_no_evil:

Reply:
That's okay. I don't have any other ideas how to make it shorter :laughing:

Reply:
"[...] on Secure Booted hosts" doesn't make any sense here. Please keep "[...] Secure Boot enabled hosts" as suggested by @goarsna.